apps/opik-backend/config.yml

```diff
@@ -80,3 +80,6 @@ usageReport:
 
 metadata:
   version: ${OPIK_VERSION:-latest}
+
+cors:
+  enabled: ${CORS:-true}
```

Comment:
I believe the default should be `false` (as it is now) and only enabling overriding explicitly when needed

Reply:
Yeah I wasn't sure on this part. I wanted it to be similar to other backend applications. Will change that.

Reply:
agree, we should disable Cross-Origin Resource Sharing by default, allow people to override it.

when CORS = false. its disabled (not allowing requests from cross origin) 
when CORS = true. its enabled (we allow you to call opik from cross origin)

Reply:
Done